packages/venia-concept/src/components/Checkout/section.js

```diff
@@ -1,33 +1,40 @@
 import React, { Component } from 'react';
 import { bool, node, shape, string } from 'prop-types';
-import CheckIcon from 'react-feather/dist/icons/check';
+import EditIcon from 'react-feather/dist/icons/edit-2';
 
 import classify from 'src/classify';
 import Icon from 'src/components/Icon';
 import defaultClasses from './section.css';
 
+const editIconAttrs = {
+    color: 'black',
```

Comment:
Can you add a TODO for moving these attributes into a stylesheet? We want to colocate interdependent things like pixel width, but we don't have an agreed-upon means of doing it yet. (I favor using a CSS Modules [:export block](https://github.com/css-modules/icss#export), but any choice will need @jimbo to weigh in, and can come after release.

Reply:
I like the idea of the export block (and the comment). Also agree that we should do it project-wide after the release.